apps/win32/package.json

```
@@ -15,10 +15,10 @@
     "code-style": "fluentui-scripts code-style",
     "depcheck": "fluentui-scripts depcheck",
     "lint": "fluentui-scripts eslint",
-    "start": "fluentui-scripts metro --server --cli --port 8081",
+    "start": "react-native rnx-bundle --server --port 8081",
```

Shouldn't this be `rnx-start`?
```suggestion
    "start": "react-native rnx-start",
```

---

Yes. :)